src/main/java/Ui.java

```diff
@@ -1,25 +1,25 @@
 import java.util.Scanner;
 
 class Ui {
-    private final String LOGO = " ____        _        \n"
+    private final String logo = " ____        _        \n"
```

Comment:
`logo` can be marked `static` and `final`, keeping the variable name uppercased.